ursina/gamepad.py

```diff
@@ -6,7 +6,17 @@
     text_entity = Text()
     player = Entity(model='cube', color=color.azure)
 
+    input_handler.gamepads = base.devices.getDevices()
+    print(gamepad.buttons)
+    print(gamepad.axes)
+
     def update():
+        for button in gamepad.buttons:
+            print(button.handle.name,":", button.pressed)
+
+        for axis in gamepad.axes:
+            print(axis.axis, ":", axis.value)
+
```

Comment:
Please remove debug prints.

Reply:
Ouch...

Reply:
If you can push force on master and remove all this mess. Sorry for causing this noise